### PR TITLE
Work page update

### LINKED ISSUE #391: Work Page - finishing touches
- [x] change according to latest design:

Work Updated

### PR DESCRIPTION
**Pull request by pichfl**
- [x] New shapes
- [x] Resize logos to similar heights

closes #391

### CODE REVIEW

**Review by marcoow — CHANGES_REQUESTED** [review]
* trainline teaser is wrong: 
Bildschirmfoto 2019-05-07 um 11 49 48
* timify teaser is wrong: 
Bildschirmfoto 2019-05-07 um 11 50 07
* process teaser has wrong text and link: 
Bildschirmfoto 2019-05-07 um 11 50 23

**Comment by pichfl** [comment]
Was a bit over-eager to get this out, will update.

**Comment by pichfl** [comment]
Updated again with correct horizontal spacing for the taglines

File: src/ui/components/ShapeAcrossPlaybook/template.hbs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<ShapeAcross @background="#222" @src="/assets/images/team-meeting.jpg" @width="7895" @height="5266">
+  <h3 class="typography.small-text offset-vertical">
+    Playbook
+  </h3>
+  <p class="typography.body-text">
+    Our process delivers sustainable solutions
```
Comment (marcoow) [inline src/ui/components/ShapeAcrossPlaybook/template.hbs]: Let's use the same text from https://deploy-preview-405--gallant-hawking-42050a.netlify.com/services here:

> We maintain a simple and client-focussed process. Find out more about how we work to deliver ambitious software solutions for our clients.

**Review by marcoow — APPROVED** [review]

**Pull request merged**